doc/introduction/measurements.rst

```diff
@@ -154,6 +154,71 @@ so corresponds to a :math:`99.75\%` probability of measuring
 state :math:`|00\rangle`, and a :math:`0.25\%` probability of
 measuring state :math:`|01\rangle`.
 
+Mid-circuit measurements and conditional operations
+---------------------------------------------------
+
+PennyLane allows specifying measurements in the middle of the circuit.
+Operations can then be conditioned on the measurement outcome of such
+mid-circuit measurements:
+
+.. code-block:: python
+
+    def my_quantum_function(x, y):
+        qml.RY(x, wires=1)
+        qml.CNOT(wires=[0, 1])
+        m_0 = qml.measure(1)
```

Comment:
UI looking awesome - do we want `qml.measure(wires=1)`, i.e., with `wires=1` to be the required style? It feels a bit more explicit and familiar with other PL circuit elements.

Reply:
My thinking here was that singe `qml.measure` accepts a single qubit, it looks better to showcase `qml.measure(1)`. We have the same shorthand available for observables (we can write `qml.expval(qml.PauliZ(0))` instead of having to write `qml.expval(qml.PauliZ(wires=0))`. `qml.measure(wires=1)` is still valid as is.